backend/danswer/one_shot_answer/answer_question.py

```diff
@@ -204,12 +213,17 @@ def stream_answer_objects(
         # tested quotes with tool calling too much yet
         skip_explicit_tool_calling=True,
         return_contexts=query_req.return_contexts,
+        skip_generation=True,
     )
-    # won't be any ImageGenerationDisplay responses since that tool is never passed in
+
+    # won't be any ImageGenerationDisplay responses since that tool is never passed in go
     dropped_inds: list[int] = []
+
     for packet in cast(AnswerObjectIterator, answer.processed_streamed_output):
         # for one-shot flow, don't currently do anything with these
         if isinstance(packet, ToolResponse):
+            # add a handling of the response here to add the information regarding relevance to the search results.
```

Comment:
What is this? There is no change here. Was this just a note to add the thing below?